index.html

```diff
@@ -0,0 +1,87 @@
+html> 
+   <head> 
+      <script> 
+         
+         function dis(val) 
+         { 
+             document.getElementById("result").value+=val 
+         } 
+           
+        
+         function solve() 
+         { 
+             let x = document.getElementById("result").value 
+             let y = eval(x) 
+             document.getElementById("result").value = y 
+         } 
+           
+          
+         function clr() 
+         { 
+             document.getElementById("result").value = "" 
+         } 
+      </script> 
+    
+      <style> 
+         .title{ 
+         margin-bottom: 10px; 
+         text-align:center; 
+         width: 210px; 
+         color:rgb(0, 0, 0); 
+         border: solid rgb(231, 10, 213) 2px; 
+         } 
+  
+         input[type="button"] 
+         { 
+         background-color:rgb(252, 151, 0); 
+         color: black; 
+         border: solid black 2px; 
+         width:100% 
+         } 
+  
+         input[type="text"] 
+         { 
+         background-color:rgb(255, 0, 0); 
+         border: solid black 2px; 
+         width:100% 
+         } 
+      </style> 
+   </head> 
+   
+   <body> 
+      <div class = title >Kolbys first calculator</div> 
+      <table border="1"> 
+         <tr> 
+            <td colspan="3"><input type="text" id="result"/></td> 
+            
+            <td><input type="button" value="c" onclick="clr()"/> </td> 
+         </tr> 
+         <tr> 
+           
+            <td><input type="button" value="1" onclick="dis('1')"/> </td> 
+            <td><input type="button" value="2" onclick="dis('2')"/> </td> 
```

Comment:
The homework is meant for you to use your skills that you learn in class.  This is from the tutorial.  It was looking for 2 boxes for numbers an operator and an equals with an answer.  I recommend using your part two to go back to the basic concepts that you learned in class last week and this week.